src/rtkpos.c

```diff
@@ -2193,73 +2228,110 @@ static int relpos(rtk_t *rtk, const obsd_t *obs, int nu, int nr,
 }
 
 /* ------------------------------------------------------------------------------ */
+/* rtk API functions */
 
-static int rtk_is_valid(const rtk_t *rtk)
+extern int rtk_is_valid(const rtk_t *rtk)
```

Comment:
I'm pretty [sure](https://stackoverflow.com/questions/1433204/how-do-i-use-extern-to-share-variables-between-source-files) you should put the declaration to header and you don't need extern here.